src/deploy.rs

```diff
@@ -78,27 +87,39 @@ impl DeployCmd {
             n += 1;
         }
 
-        self.print_ansible_run_banner(n, total, "Provision Genesis Node");
-        self.provision_genesis_node().await.map_err(|err| {
-            println!("Failed to provision genesis node {err:?}");
-            err
-        })?;
-        n += 1;
+        let this_is_a_new_network = self.bootstrap_peer.is_none();
+        let initial_point_of_contact = if let Some(contact) = &self.bootstrap_peer {
+            println!("Using bootstrap peer: {contact}, waiting 60s for initital inventory spin up");
+            sleep(Duration::from_secs(60)).await;
 
-        let (genesis_multiaddr, _) = get_genesis_multiaddr(
-            &self.testnet_deploy.ansible_runner,
-            &self.testnet_deploy.ssh_client,
-        )
-        .await
-        .map_err(|err| {
-            println!("Failed to get genesis multiaddr {err:?}");
-            err
-        })?;
-        println!("Obtained multiaddr for genesis node: {genesis_multiaddr}");
+            contact.clone()
+        } else {
+            self.print_ansible_run_banner(n, total, "Provision Genesis Node");
+            self.provision_genesis_node().await.map_err(|err| {
+                println!("Failed to provision genesis node {err:?}");
+                err
+            })?;
+            n += 1;
```

Comment:
A tiny nit pick, but if `!is_fresh_network` then `total` has to be updated. It just prints the max steps in the banner.

Reply:
i'm not sure what you mean to change here.

Reply:
the total is being updated, no? with `n+=1` ?